src/content/docs/fr/2/guide/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -0,0 +1,1144 @@
+- `Plugin::setup_with_config` enlevé. Utilisez `tauri::Plugin::PluginApi` mis à jour à la place.
+- `scope::ipc::RemoteDomainAccessScope::enable_tauri_api` et `scope::ipc::RemoteDomainAccessScope::enables_tauri_api` enlevés. Activer chaque plugin "core" individuellement via `scope::ipc::RemoteDomainAccessScope::add_plugin` à la place.
+- `updater` module enlevé. Utilisez `tauri-plugin-updater` à la place. [Migration](#migrer-vers-un-updater-plugin)
+- `Menu`, `MenuEvent`, `CustomMenuItem`, `Submenu`, `WindowMenuEvent`, `MenuItem` et `Builder::on_menu_event` APIs enlevés. [Migration](#migrer-vers-un-menu)
+- `SystemTray`, `SystemTrayHandle`, `SystemTrayMenu`, `SystemTrayMenuItemHandle`, `SystemTraySubmenu`, `MenuEntry` et `SystemTrayMenuItem` APIs enlevées. [Migration](#migrer-vers-un-tray-icon-module)
+
+### Changements de l'API JavaScript
+
+Le package `@tauri-apps/api` ne fournis plus de modules "non-core". Seul les modules `tauri`, `path` et `event` sont exportés. Tous les autres ont été déplacés vers des plugins.
+
+- `@tauri-apps/api/app` module enlevé. Utilisez `@tauri-apps/plugin-app` à la place. [Migration](#migrer-votre-app-plugin)
+- `@tauri-apps/api/cli` module enlevé. Utilisez `@tauri-apps/plugin-cli` à la place. [Migration](#migrer-vers-un-cli-plugin)
+- `@tauri-apps/api/clipboard` module enlevé. Utilisez `@tauri-apps/plugin-clipboard` à la place. [Migration](#migrer-vers-un-clipboard-plugin)
+- `@tauri-apps/api/dialog` module enlevé. Utilisez `@tauri-apps/plugin-dialog` à la place. [Migration](#migrer-vers-un-dialog-plugin)
+- `@tauri-apps/api/fs` module enlevé. Utilisez `@tauri-apps/plugin-fs` à la place. [Migration](#migrer-vers-un-file-system-plugin)
+- `@tauri-apps/api/global-shortcut` module enlevé. Utilisez `@tauri-apps/plugin-global-shortcut` à la place. [Migration](#migrer-vers-un-global-shortcut-plugin)
+- `@tauri-apps/api/http` module enlevé. Utilisez `@tauri-apps/plugin-http` à la place. [Migration](#migrer-vers-un-plugin-http)
+- `@tauri-apps/api/os` module enlevé. Utilisez `@tauri-apps/plugin-os` à la place. [Migration](#migrer-vers-un-os-plugin)
+- `@tauri-apps/api/notification` module enlevé. Utilisez `@tauri-apps/plugin-notification` à la place. [Migration](#migrer-vers-un-notification-plugin)
+- `@tauri-apps/api/process` module enlevé. Utilisez `@tauri-apps/plugin-process` à la place. [Migration](#migrer-vers-un-process-plugin)
+- `@tauri-apps/api/shell` module enlevé. Utilisez `@tauri-apps/plugin-shell` à la place. [Migration](#migrer-vers-un-shell-plugin)
+- `@tauri-apps/api/updater` module enlevé. Utilisez `@tauri-apps/plugin-updater` à la place [Migration](#migrer-vers-un-updater-plugin)
+- `@tauri-apps/api/window` module enlevé. Utilisez `@tauri-apps/plugin-window` à la place [Migration](#migrer-vers-un-window-plugin)
+
+## Etapes de Migration Détaillées
+
+Les scénarios communs que vous pouvez rencontrer lors de votre migration de Tauri 1.0 vers Tauri 2.0.
+
+### Migrer votre "App Plugin"

```

Comment:
Should "App Plugin" be in quotes here I wonder? Similar for all following headings

Reply:
I put some quotes here because i can't translate them without loosing the meaning, in this way the reader is not confuse to find some non-French in the sentence. But again, the reader will understand, even more because he probably know why he is here for.

Reply:
Is there a translation for "Plugin" that would work? Because then you could do `Merger votre "App" [French word for plugin]`

Reply:
And the same would apply to the additional headings below.

Reply:
This is the problem, there is no French equivalent for "plugin" his definition, that's why everybody who need to talk about it says "plugin" but it is not in the french dictionnary. Your solution makes sens, `Mirger votre "App" plugin` is understandable. However since you're out the quotes, you need to apply the french rules and invert the words and put `de` between which would makes : `Migrer votre plugin d'"App"` and actually looses the meaning at the same time. If it still understandable for this one, some like `Migrer vers un plugin de "Clipboard"` or even translate clipboard like `Migrer vers un plugin de presse-papier" is really unthinkable because it really makes no sens.

Reply:
I'll leave it to you since you're much more the expert, but I think my preference would be "Migrer votre App Plugin" and to think of "App Plugin" as a proper noun in a way (especially since the package names are `@tauri-apps/plugin-app` for example).